playbooks/dependencies.md

```diff
@@ -0,0 +1,91 @@
+within Artsy. E.g. A dependency added to palette (our design system primitives) ends up in many repos transitively.
+
+This policy can be **less** conservative when it’s not used outside in production at runtime. Like, task runners,
+developer tools, linters or CI steps. Issues around these tools won’t block daily work and can be removed in
+exchange for some kind of tech debt.
+
+This document aims to emphasise the downsides of dependencies; but there are a lot of positives! We should aim for
+"Proudly found elsewhere" to ensure we don’t duplicate existing work and help improve projects for everyone. We
+want to be adding thoughtfully, not blocking any new dependencies.
+
+## Why?
+
+Dependencies help you go from `A -> B` faster, maybe `B` is a working in-app modal popup which acts like how you’d
+expect across many browsers or operating systems. There are trade-offs to taking on a dependency, though:
+
+- They can exist outside of your control _( The maintainer may have moved on, transitive dependencies )_
+
+- They can block you further down the line in unexpected ways _( Your animation library may only work on one
+  platform, or may make updating another dependency harder because they use older APIs )_ which can make further
+  migrations hard
+
+- They can provide a lot more code than you need _( You only need a single function which lives inside a whole
+  framework )_ every line of code you add is code we need to maintain, which again can make migrations hard ( The
+  code we don’t use may be blocking an upgrade )
+
+- More dependencies means more code churn _( Their APIs may change across versions, and if you don’t keep
+  up-to-date then larger jumps are harder. They may have security vulnerabilities which means you have to update
+  that dependencies and get any transitives in sync )_
+
+## A rubric on whether a dependency needs an RFC?
+
+If you answer yes to most of these questions, then you probably don’t need to write an RFC
+
+- Could you fit this codebase in your head after reading the source?
+- Is this the stand-out obvious answer to a particular domain problem?
+- Do you expect your team to be the only people who know about this dependency?
+- Is this obviously being used in production by the maintainers, and thus battle-tested?
+- Are the dependencies of this dependency already in our projects, or is the dependency itself a transitive
+  dependency of another dependency we already rely on?
+- Would this dependency be the first time we’ve needed something of this domain?
+- Do you feel well versed in the domain of this dependency and/or could maintain it if that needs to become an
+  option?
+
+## Ways to safely work around this
+
+- Re-use existing dependencies, or build the exact implementation that addresses just our needs today. Rather than
+  trying to foresee the future, we should re-asses once we have more information.
```

Comment:
I'm curious if we can word this around ["Build for 10x"](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#build-for-10x)